connect/file/src/main/java/org/apache/kafka/connect/file/FileStreamSourceConnector.java

```diff
@@ -43,7 +43,7 @@
 
     private static final ConfigDef CONFIG_DEF = new ConfigDef()
         .define(FILE_CONFIG, Type.STRING, null, Importance.HIGH, "Source filename. If not specified, the standard input will be used")
-        .define(TOPIC_CONFIG, Type.STRING, Importance.HIGH, "The topic to publish data to")
+        .define(TOPIC_CONFIG, Type.LIST, Importance.HIGH, "The topic to publish data to")
```

Comment:
I think using a string and the `NonEmptyString` validator would be a little clearer. That would allow us to skip any additional checks in `start`.

Reply:
Problem is that we check for `,` to avoid multiple topics. Which I find not great. Parsing it as a list will allow us to do both.

Reply:
Ack, thanks for the explanation.